students/charlie_england/lesson05/assignment/tests/test_database.py

```diff
@@ -3,46 +3,51 @@
 """
 
 import os
+import sys
 import pytest
-
+sys.path.insert(0,'../src')
```

Comment:
Similar to before, be careful when using sys.path, you don't want to always modify it if you can figure out how to run it from the same valid location